setup.py

```diff
@@ -249,16 +297,27 @@ def main():
         clean()
         sys.exit()
 
+    libraries = []
+    static_lib_dir = j("bblfsh", "libuast")
+    static_libraries = ["{}/libuast".format(static_lib_dir)]
+
+    if sys.platform == 'win32':
+        libraries.extend(static_libraries)
+        libraries.extend(["legacy_stdio_definitions", "winmm", "ws2_32"])
+        extra_objects = []
+    else:  # POSIX
+        extra_objects = ['{}.a'.format(l) for l in static_libraries]
+
     libuast_module = Extension(
         "bblfsh.pyuast",
         libraries=libraries,
-        library_dirs=["/usr/lib", "/usr/local/lib"],
         extra_compile_args=["-std=c++11"],
-        include_dirs=[j("bblfsh", "libuast"), "/usr/local/include", "/usr/local/include/libxml2",
-                      "/usr/include", "/usr/include/libxml2"], sources=sources)
+        extra_objects=extra_objects,
+        include_dirs=[j("bblfsh", "libuast")],
+        sources=sources)
 
     setup(
-        cmdclass = {
+        cmdclass={
```

Comment:
I believe pep8 want to see spaces here. :)

Reply:
I don't think so, removing the spaces gives on my Vim PEP8 checker:

`E251: Unexpected spaces around keyword / parameter equals`

Reply:
oh, sure. sorry.

Reply:
Drive-by: Do we (or if not then could we) use a formatter like [yapf](https://github.com/google/yapf), which support PEP8 style?

Reply:
I belive ML team [might be using flake8](https://github.com/search?q=org%3Asrc-d+pep8&type=Issues)

Reply:
As far as I know, though, flake8 only _complains_ about errors, it doesn't fix them. For formatting questions, it'd be nice to have a tool that does the right thing.